03-data-frames.Rmd

```diff
@@ -18,9 +18,11 @@ surveys <- read.csv("data/portal_data_joined.csv")
 
 > ## Learning Objectives
 >
-> * understand the concept of a `data.frame`
-> * use sequences
-> * know how to access any element of a `data.frame`
+> * describe what a `data.frame` is
+> * read data from a file into a `data.frame` and change how character strings are handled
+> * summarize the size and data types of a `data.frame` 
+> * write a command to print odd numbers from 90 to 100
```

Comment:
that might be a little too specific. The goal here is for learners to be able to create sequences of their own.